packages/react/src/patterns/sections/CardSection/CardSection.js

```diff
@@ -87,7 +87,8 @@ const _renderCards = (cards, containerRef, cta) => (
     ref={containerRef}>
     {cards.map((card, index) => {
       return (
-        <div className={`${prefix}--card-section__cards__col`}>
+        <div
+          className={`${prefix}--card-section__cards__col ${prefix}--card-section__card`}>
```

Comment:
Small thing, but should `card-section__card` be passed in as a `customClassName` for the `Card`?